templates/sections/certifications.html

```
@@ -0,0 +1,24 @@
+    {{TITLE}}      — certification name (HTML-escaped)
+    {{ORG_BLOCK}}  — conditional: rendered when org is present, otherwise an empty
+                      <span class="cert-org"></span> is emitted for table-cell alignment.
+                      Contains {{ORG}}.
+    {{ORG}}        — issuing organization (HTML-escaped, inside ORG_BLOCK)
+    {{YEAR_BLOCK}} — conditional: same pattern as ORG_BLOCK for cert-year alignment.
+                      Contains {{YEAR}}.
+    {{YEAR}}       — year issued (HTML-escaped, inside YEAR_BLOCK)
+
+  {{ITEMS}} receives the concatenated rendered entries.
+-->
+<div class="cert-item">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)